wagtailio/utils/models.py

```diff
@@ -143,3 +143,44 @@ class CrossPageMixin(models.Model):
 
     class Meta:
         abstract = True
+
+
+class SVGIcon(models.TextChoices):
```

Comment:
suggestion: move this in its own module (`choices.py`).
Please liaise with @William-Blackie for the icon list. Ref: #181

Reply:
done in d53e781e9a9dd0d87e4185fc156dcedbab6f4e74